### PR TITLE
e2e: performance: update path of latency binary

### PR DESCRIPTION
**Pull request by shajmakh**
Signed-off-by: shereenH <shajmakh@redhat.com>

### CODE REVIEW

File: test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go [hunk 1]
```diff
@@ -15,7 +15,9 @@ import (
 	testlog "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/log"
 )
 
+//TODO get commonly used variables from one shared file that defines constants
```
Comment (yanirq) [inline test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go:18]: Do we want to leave this TODO ?

Comment (shajmakh) [reply test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go:18]: yes, this will be an enhancement for the tests

**Pull request merged**